File: Other/tasks/2_remove_from_string.py
#2) Write a method which will remove any given character from a String?

def start():

    while True:
        userInput = input ("Input a string and a comma and a character to be removed. X to E(X)it.")
        if userInput.upper() == "X":
            print("Good bye")
            break
        else:
            userInput = userInput.split(",")
            result = removal(userInput[0], userInput[1])
            print("word {0} is {1} without {2}".format( userInput[0],result, userInput[1]))


def removal(word, letter):
    removed = ""
    for x in word:
        if x == letter:
            continue
        else:
            removed = removed + x

    print (removed)
    return removed





if __name__ == "__main__":
    start()

File: Other/tasks/2_remove_from_string.php
//2) Write a method which will remove any given character from a String?

<?php
/* Write your PHP code here */
//2) Write a method which will remove any given character from a String?

$word = "removaaaae a aaaa letaateraaa";
$letter = "a";
function removal($word, $letter){
	$result = "";
	//$word = str_split($word);

	//echo is_array($word) ? 'it is an array' : 'not an Array';



	for ($i = 0; $i < strlen($word); $i++){
		//echo $word[$i]." <br\> ";

		if ($word[$i] != $letter){
			//echo $word[$i];
			$result .= $word[$i];
		}

	}

	echo $result;
}

removal($word, $letter);
?>
